e2e/inference/README.md

```diff
@@ -1,12 +1,14 @@
 ## Intel AI Inference E2E Solution for OpenShift
 
 ### Overview
-Intel AI inference e2e solution for OCP is built upon Intel® dGPU provisioning for OpenShift and Intel® Xeon® processors. Two AI inferencing modes are supported:
+Intel AI inference e2e solution for OCP is built upon Intel® dGPU provisioning for OpenShift and Intel® Xeon® processors. The two following AI inference modes are used to test with the Intel Data Center GPU Card provisioning:
 * **Interactive Mode**
-[Open Data Hub (ODH)](https://github.com/opendatahub-io) and [Red Hat OpenShift Data Science (RHODS)](https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-data-science) provides Intel OpenVINO™ based [Jupiter Notebook](https://jupyter.org/) to help users interactively debug the inferencing applications or optimize the models with OCP using Intel Data Center GPU cards and Intel Xeon processors.
+[Open Data Hub (ODH)](https://github.com/opendatahub-io) and [Red Hat OpenShift Data Science (RHODS)](https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-data-science) provides Intel® OpenVINO™ based [Jupiter Notebook](https://jupyter.org/) to help users interactively debug the inferencing applications or optimize the models with OCP using Intel Data Center GPU cards and Intel Xeon processors.
```

Comment:
one typo Jupiter to Jupyter

Reply:
Since it is only one typo, how about we merge this now, and send another PR to fix the typo?